src/lastlog.c

```diff
@@ -150,9 +150,12 @@ static void print_one (/*@null@*/const struct passwd *pw)
 
 	ll_time = ll.ll_time;
 	tm = localtime (&ll_time);
-	strftime (ptime, sizeof (ptime), "%a %b %e %H:%M:%S %z %Y", tm);
-	cp = ptime;
-
+	if (tm == NULL) {
+		cp = "(unknown)";
```

Comment:
Do you think it's worth doing _("(unknown)") ?

Reply:
I don't mind changing the message but we definitely need to set something, as `cp` will be printed several lines below.

Reply:
All right well I'll merge it as is for now, thanks.